client/app/components/navigation/Login.tsx

```diff
@@ -15,23 +16,27 @@ export default function Login() {
 
   return (
     <>
-      <Link href="/dashboard?role=admin" sx={{ color: "white" }}>
+      <Link href="#" sx={{ color: "white" }}>
         <Button
           sx={{ ...commonButtonConfig }}
           aria-label="Program Administrator Log In"
           color="inherit"
           variant="outlined"
+          onClick={() => signIn("keycloak", undefined, { kc_idp_hint: "idir" })}
         >
           Program Administrator
           <br /> Log In
         </Button>
       </Link>
-      <Link href="/dashboard?role=operator" sx={{ color: "white" }}>
+      <Link href="#" sx={{ color: "white" }}>
         <Button
           sx={{ ...commonButtonConfig, marginLeft: "20px" }}
           aria-label="Industrial Operator Log In"
           color="inherit"
           variant="outlined"
+          onClick={() =>
+            signIn("keycloak", undefined, { kc_idp_hint: "bceidboth" })
```

Comment:
Restricting the path to business bceid only
```suggestion
            signIn("keycloak", undefined, { kc_idp_hint: "bceidbusiness" })
```

Reply:
Thanks! :)